editor.planx.uk/src/lib/graphql.ts

```diff
@@ -83,7 +72,19 @@ const retryLink = new RetryLink({
   },
 });
 
+/**
+ * Client used to make all requests by authorised users
+ */
 export const client = new ApolloClient({
```

Comment:
I felt keeping the simple `client` name was probably more appropriate and concise compared to `authorisedClient` or similar. This will be used for almost all frontend calls as most of the complexity is within the Editor.